docker/docker-compose-centos-dev.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, EMC, Inc.
```

Comment:
question to @zhangy42 , what does ```docker/docker-compose-centos.yml``` be used for ? I didn't find it anywhere in the readme.md ?